scripts/tipitaka-import/TIPITAKA_IMPORT.md

```
@@ -69,46 +65,47 @@ export async function runTipitakaImport(): Promise<void> {
-		const segmentTypeIdMap = new Map(
-			segmentTypes.map((item) => [item.key as SegmentType["key"], item.id]),
-		);
-		const primaryTypeId = segmentTypeIdMap.get("PRIMARY");
-		if (!primaryTypeId) {
-			throw new Error('Segment type "PRIMARY" not found');
-		}
+**ファイル**: `run.ts` (19-21行目), `db/segment-types.ts`, `db/metadata-types.ts`, `initial-setup/root-page.ts`
 
-		const rootPage = await ensureRootPage(prisma, user.id, primaryTypeId);
+```19:21:scripts/tipitaka-import/run.ts
+		// Step 1: セグメントタイプ、メタデータタイプ、ルートページの初期セットアップ
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)